### PR TITLE
fix: allow project path to wrap, adjust css to avoid truncated scroll area

### LINKED ISSUE #6064: bug: long path can break its container on get started widget
## Describe the bug

Screen Shot 2021-03-01 at 4 03 48 PM

### LINKED ISSUE #6065: bug: can't scroll to bottom of settings when get started is visible
## Describe the bug

With the feature flag enabled, i cannot see the "Delete bot" button at the bottom of the settings. I can't scroll down far enough for some reason.

I believe this is due to a calculated height CSS property that may need to be adjusted

## Version

nightly

ZlhD4Ly0i4

### LINKED ISSUE #6084: Getting Started - Bot Info exceeds its cell width in UI
## Describe the bug

The Bot Info if text exceeds the length of the cell for the Bot Name and file location does not stay within cell boundries.

image

## Version

image

## Browser

- [X] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [X] Windows
- [ ] Ubuntu

## To Reproduce

1. Create Bot with a really long name and long path
2. Go to Bot Settings page and look in Bot Info Section

## Expected behavior

Text is contained within cell / wrapped or concatenated

### PR DESCRIPTION
**Pull request by benbrown**
## Description

* allow the project path on the get started widget to wrap if long
* adjust Css on the page template so that when get started is visible, the scroling area below is not truncated

## Task Item

fixes #6084 
fixes #6065 
fixes #6064

## Screenshots

Screen Shot 2021-03-02 at 11 17 13 AM

### CODE REVIEW

File: Composer/packages/client/src/components/GetStarted/GetStarted.tsx [hunk 1]
```diff
@@ -120,7 +120,17 @@ export const GetStarted: React.FC<GetStartedProps> = (props) => {
         <Stack.Item grow={0} styles={{ root: { width: 250, borderRight: '1px solid #3d3d3d', padding: 38 } }}>
           <h2 style={h2Style}>{botProject?.name}</h2>
           <p>
-            {formatMessage('File Location:')} <span style={{ textOverflow: 'ellipses', fontSize: 12 }}>{location}</span>
+            {formatMessage('File Location:')}{' '}
```
Comment (hatpick) [inline Composer/packages/client/src/components/GetStarted/GetStarted.tsx:123]: nit: maybe include the space in the formatMessage

File: Composer/packages/client/src/components/Page.tsx [hunk 2]
```diff
@@ -81,7 +81,7 @@ export const main = (hasRenderHeaderContent) => css`
 
 export const mainWithGetStarted = css`
   margin-left: 2px;
-  height: calc(100vh - 385px);
+  height: calc(100vh - 517px);
```
Comment (hatpick) [inline Composer/packages/client/src/components/Page.tsx:84]: this seems random, can you extract it to its own variable with a descriptive name?

Comment (benbrown) [reply Composer/packages/client/src/components/Page.tsx:84]: ok!

File: Composer/packages/client/src/components/GetStarted/GetStarted.tsx (hunk 1 above)
Comment (benbrown) [reply Composer/packages/client/src/components/GetStarted/GetStarted.tsx:123]: this was unnecessary i removed it.

**Comment by benbrown** [comment]
@hatpick when you get a chance, can you re-check this please

**Review by hatpick — APPROVED** [review]

**Pull request closed without merging**